File: Python Word examples/tables/add_table_row.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
folder = "input"
dest_file_path = "output/Tables/add_table_row.docx"
table_path = "sections/0/tables/0"
result = api.add_table_row(filename, table_path, folder, dest_file_path, index_of_table_row=5, cells_count=2)

File: JavaScript Word Examples/TablesApiDemo/Rows/addTableRow.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "Table.docx";
    var tablePath = "Section/0/Body/0/Table/0";
    var opts = {
        folder: 'Tables',
        indexOfTableRow: 0,
        cellsCount: 4
    };
    var desFilePath = "output/Tables/AddTableRow_1.docx";
    instance.addTableRow(filename, tablePath, desFilePath,opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();